.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,308 @@
+          echo 'Description=ODBC Driver 12 for PostgreSQL Server' | sudo tee -a /etc/odbcinst.ini > /dev/null
+          echo 'Driver=/usr/local/lib/psqlodbcw.so' | sudo tee -a /etc/odbcinst.ini > /dev/null
+          echo 'UsageCount=1' | sudo tee -a /etc/odbcinst.ini > /dev/null
+      
+      - name: Run ODBC Tests
+        if: always() && steps.install-sql-server-odbc-driver.outcome == 'success' && steps.install-psql-odbc-driver.outcome == 'success'
+        run: |
+          cd test/odbc
+          cmake -S . -B build
+          cmake --build build
+          MSSQL_ODBC_DRIVER_NAME="ODBC Driver 17 for SQL Server" \
+            MSSQL_BABEL_DB_SERVER=localhost \
+            MSSQL_BABEL_DB_PORT=1433 \
+            MSSQL_BABEL_DB_USER=jdbc_user \
+            MSSQL_BABEL_DB_PASSWORD=12345678 \
+            MSSQL_BABEL_DB_NAME=master \
+            PSQL_ODBC_DRIVER_NAME=ODBC_Driver_12_for_PostgreSQL \
+            PSQL_BABEL_DB_SERVER=localhost \
+            PSQL_BABEL_DB_PORT=5432 \
+            PSQL_BABEL_DB_USER=jdbc_user \
+            PSQL_BABEL_DB_PASSWORD=12345678 \
+            PSQL_BABEL_DB_NAME=jdbc_testdb \
+            ./build/main
+
+      - name: Install Python
+        id: install-python
+        if: always() && steps.install-extensions.outcome == 'success'
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Configure Python Environment
+        id: configure-python-environment
+        if: always() && steps.install-python.outcome == 'success'
+        run: |
+          cd ~
+          curl https://packages.microsoft.com/config/ubuntu/20.04/prod.list | sudo tee /etc/apt/sources.list.d/mssql-release.list
+          cd ~/work/babelfish_extensions/babelfish_extensions/test/python
+          mkdir sqltoolsservice
+          cd sqltoolsservice
+          wget https://github.com/microsoft/sqltoolsservice/releases/download/4.4.0.12/Microsoft.SqlTools.ServiceLayer-rhel-x64-net6.0.tar.gz && tar -xzvf Microsoft.SqlTools.ServiceLayer-rhel-x64-net6.0.tar.gz
+          cd ../
+          sudo ACCEPT_EULA=Y apt-get install -y msodbcsql17 python3-dev
+          pip3 install pyodbc==4.0.35 pymssql pytest pytest-xdist
+      
+      - name: Run Python Tests
+        if: always() && steps.configure-python-environment.outcome == 'success'
+        run: | 
+          cd test/python
+          compareWithFile=true \
+            driver=pyodbc \
+            runInParallel=false \
+            testName=all \
+            provider="ODBC Driver 17 for SQL Server" \
+            fileGenerator_URL=localhost \
+            fileGenerator_port=1433 \
+            fileGenerator_databaseName=master \
+            fileGenerator_user=jdbc_user \
+            fileGenerator_password=12345678 \
+            pytest -s --tb=long -q .
+
+      - name: Generate code coverage HTML report
+        id: code-coverage
+        if: always()
+        run: |
+          export PG_CONFIG=~/psql/bin/pg_config
```

Comment:
Let's do this in a for loop for all the 4 extensions to avoid code duplication

Reply:
Done.